images/README.md

```diff
@@ -1,15 +1,11 @@
 # Dev Images
 
-This folder contains docker files for development. To build them execute `./build`. This will produce 2 following images:
+This folder contains a docker-compose file for development. To create and start the service execute `docker-compose up -d`.
```

Comment:
Remove the `docker-compose up -d` and the image list. Just explain for what this docker-compose is